### ISSUE TITLE
What about some DDEV support?

### ISSUE THREAD

**Issue by bmartinez287** [lagoon-examples/drupalnext-base#74]
it be great if the base would have support for it.

**Comment by tobybellwood** [comment]
We _shouldn't_ need to bake DDEV support into every example - as the docs at https://ddev.readthedocs.io/en/stable/users/providers/lagoon/#usage

(however, I've just tried to follow the docs, and got nowhere fast, so @rocketeerbkw and I will look into that)

**Comment by bmartinez287** [comment]
Yeah, the provider docs need some work as well. Some community effort was put into Platform.sh and Acquia but Pantheon and Lagoon need major improvements. It be great if the foundation could pattern up with Amazee.io to support it and improve it. Likely cheaper than dedicating an engineer on staff to fix it but either one would be great contribution.

I ran through DDEV hosting provider's setup the other day and although it works, it does not represent the quality and effort hosting providers have put into their systems.
The lagoon one is interesting since you have to provide a Github URL and that's how landed on the lagoon examples.
